git/git.go

```diff
@@ -7,7 +7,6 @@ import (
 	"io"
 	"os/exec"
 	"strings"
-
 	"github.com/github/git-lfs/vendor/_nuts/github.com/rubyist/tracerx"
```

Comment:
Can you please add the empty line again?  This is the import formatting done by [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports).  If you have it installed, you can use `script/fmt` to format the git-lfs code.

Here's how you can update this PR (replace editor with the editor of your choice; only needed if you don't have goimports):
```
$ editor git/git.go
$ script/fmt
$ git add --patch git/git.go
$ git commit --amend
$ git push <remote> <localbranch>:blinking
```